src/content/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability.mdx

```diff
@@ -64,9 +67,17 @@ Ready to get started? Find your quickstart in New Relic I/O:
 </ButtonLink>
 </ButtonGroup>
 
-Alternatively, if you're comfortable with the command line, our guided install discovers the applications, infrastructure, and log sources running in your environment, and recommends which ones should be instrumented. The install automates the configuration and deployment of each system you choose to instrument.
+New Relic I/O is open source, which means that you can modify and improve existing quickstarts, or build new ones, to suit your needs. We thoroughly review external edit to our quickstarts for value and quality. Interested in contributing to the community? Check out our [contributor guide](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md).
```

Comment:
I think s/be "external edits" plural
Also recommend "contributor guide in GitHub" so they know where link sends them

Reply:
I haven't added the link to the contributor guide because there's a jira to consolidate them, will add the link once we have a final doc.

Reply:
Agreed.